account_invoice_report_ddt_group/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
La licenza del modulo in questo momento è GPL, quindi questa parte dovrebbe essere modificata di conseguenza